slides/10_SideEffects.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"
+    />
+
+    <title>10 SideEffects | Essential React</title>
```

Comment:
In my opinion this chapter title (here, heading, and file name) should be "Side Effects" (separated by space). It might confuse otherwise, since it is not an existing component or concept explicitly named like this.